probatus/utils/shap_helpers.py

```diff
@@ -200,3 +201,66 @@ def calculate_shap_importance(shap_values, columns, output_columns_suffix=""):
     importance_df = importance_df.sort_values(f"mean_abs_shap_value{output_columns_suffix}", ascending=False)
 
     return importance_df
+
+
+def get_shap_cv_values(model, X_train, y_train, cv, approximate=False, check_additivity=True, **shap_kwargs):
+    """
+    Calculates SHAP values of a model via cross validation.
+
+    Args:
+        model (binary model):
+            Trained model.
+
+        X_train (pd.DataFrame or np.ndarray):
```

Comment:
I would just call it X and y. Then the train folds fan be called X_train and test folds X_test. 
That is how we called it in other modules e.g. ShapRFECV. It makes sense,  because you can run this CV on any dataset, X_train or all the data you have, just to get an understanding of the dataset.